bitnami/consul/Chart.yaml

```diff
@@ -23,4 +23,4 @@ name: consul
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/consul
   - https://www.consul.io/
-version: 10.8.2
+version: 10.9.2
```

Comment:
Thanks for the PR, please note there is an issue with the version bump; it should be `10.9.0`

Reply:
Thanks, changed it to 10.9.0